File: 0217_contains-duplicate.clj
;; https://leetcode.com/problems/contains-duplicate/

;; solution 1
(defn contain-duplicate [nums]
  (not=
   (->> nums count)
   (->> nums set count)))
(def trial1 (contain-duplicate [1 2 3 4]))
(def trial2 (contain-duplicate [1 2 3 4]))

File: 0217_contains-duplicate.py
# https://leetcode.com/problems/contains-duplicate/

# solution 1
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        return len(set(nums)) != len(nums)
trial = Solution()
trial.containsDuplicate([1,2,3,1])
trial.containsDuplicate([1,2,3,4])
trial.containsDuplicate([1,1,1,3,3,4,3,2,4,2])